spec/erb_lint/cli_spec.rb

```diff
@@ -51,7 +51,11 @@ def run(_processed_source)
         expect { subject }.to(output(/erblint \[options\] \[file1, file2, ...\]/).to_stderr)
       end
 
-      it 'shows all known linters in stderr' do
+      it 'runs load_custom_linters when showing all known linters in stderr' do
+        expect(ERBLint::LinterRegistry).to(receive(:load_custom_linters))
```

Comment:
This is understandably cheeky, but loading of linters is completely stubbed in this test 🙈